packages/app-project/src/screens/ProjectHomePage/components/MessageFromResearcher/MessageFromResearcherContainer.js

```diff
@@ -11,62 +9,39 @@ function storeMapper (stores) {
   }
 }
 
-class MessageFromResearcherContainer extends Component {
-  getMessage () {
-    return this.props.project.researcher_quote
-  }
-
-  getResearcher () {
-    const { project } = this.props
-    const researcherId = project.configuration.researcherID
-    const isUserId = /^\d+$/.test(researcherId)
-    if (isUserId) {
-      const user = project.owners.find(owner => owner.id === researcherId)
-      if (user) {
-        return user
-      }
+function getResearcher ({ project }) {
+  const researcherId = project.configuration.researcherID
+  const isUserId = /^\d+$/.test(researcherId)
+  if (isUserId) {
+    const user = project.owners.find(owner => owner.id === researcherId)
+    if (user) {
+      return user
     }
-    return undefined
-  }
-
-  getResearcherAvatar () {
-    const { project } = this.props
-    const researcher = this.getResearcher()
-    let avatar
-
-    if (project.researcher_quote) {
-      avatar = researcher
-        ? get(researcher, 'avatar_src')
-        : get(project, 'avatar.src')
-    }
-
-    return avatar
-  }
-
-  getResearcherName () {
-    const researcher = this.getResearcher()
-    return (researcher && researcher.display_name)
-      ? researcher.display_name
-      : this.props.project.display_name
-  }
-
-  getTalkLink () {
-    const { project } = this.props
-    return `/projects/${project.slug}/talk`
   }
+  return undefined
+}
 
-  render () {
-    const { researcher_quote } = this.props.project
+function MessageFromResearcherContainer({ project }) {
+  let message = project.researcher_quote
+  let researcher = getResearcher({ project })
+  let avatar = (message)
+    ? (researcher?.avatar_src)
+      ? researcher.avatar_src
+      : project.avatar.src
```

Comment:
[mumbling to self] I'm trying to think if this could ever crash... lessee... if avatar is null/undefined, then we're screwed. But in `stores/Project.js`, the model states that `avatar: types.frozen({})` so it should _always_ be an object... OK, no worries them.